src/Selenium2Driver.php

```diff
@@ -656,12 +656,18 @@ public function setValue($xpath, $value)
         $value = strval($value);
 
         if (in_array($elementName, array('input', 'textarea'))) {
-            $existingValueLength = strlen($element->attribute('value'));
+            $deleteSequence = Key::CONTROL . 'a' . KEY::NULL_KEY;
```

Comment:
Not sure. Maybe Selenium itself has reliable OS detection methods, that doesn't involve checking user agent string, that can be faked by setting corresponding header.